webapp/src/ts/effects/contacts.effects.ts

```diff
@@ -105,6 +108,8 @@ export class ContactsEffects {
       exhaustMap(([, selectedContact, forms]) => {
         return from(this.contactViewModelGeneratorService.loadReports(selectedContact, forms)).pipe(
           map(reports => {
+            const selected = { ...selectedContact, reports };
```

Comment:
I don't think we should treat effects like reducers. 
We shouldn't "reduce" the selected contact here. 
I think `loadSelectedContactTargetDoc` should get no parameters.